react/components/LogoutCustomerSession.tsx

```diff
@@ -49,19 +53,39 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm()
 
-  const getClientName = (client: any) =>
-    !!client
-      ? client.name.includes('null')
-        ? client.email.slice(0, client.email.indexOf('@'))
-        : client.name
+  const { id } = orderForm
+
+  const [customerClass, setCustomerClass] = useState('')
+
+  useEffect(()=> {
+    function getCustomerClass() {
+      const requestOptions = {
+        method: 'GET',
+      }
+      return fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions)
+    }
+  
+    getCustomerClass().then(response => response.json()).then(data => {
+      if(!!data?.clientProfileData?.customerClass) {
+        setCustomerClass(data.clientProfileData.customerClass)
+      }
+    })
+  }, [])
+
+  const getClientName = (user: any) =>
+    !!user
+      ? user.name.includes('null')
+        ? user.email.slice(0, user.email.indexOf('@'))
+        : user.name
```

Comment:
Was there any reason to change it to `user`?

Reply:
When I used this command `tslint -c tslint.json components/LogoutCustomerSession.tsx --fix`
inside react folder I was getting following error
![no-shadow-variable](https://user-images.githubusercontent.com/76150710/164381021-a9db25ad-043d-4bb3-b083-119091c7ae3e.PNG)

So I changed **client** to **user** & **mobile** to **isMobile**

Reply:
@lariciamota 
When I used this command `tslint -c tslint.json components/LogoutCustomerSession.tsx --fix`
inside react folder I was getting following error

![no-shadow-variable](https://user-images.githubusercontent.com/76150710/164381021-a9db25ad-043d-4bb3-b083-119091c7ae3e.PNG)

So I changed **client** to **user** & **mobile** to **isMobile*